### PR TITLE
fix(STONEINTG-1054):add wait for clamav db files to exist

### PR DESCRIPTION
**Pull request by chipspeak**
# Before you complete this pull request ...

Look for any open pull requests in the repository with the title "e2e-tests update" and 
see if there are recent e2e-tests updates that will be applicable to your change.

### CODE REVIEW

File: task/clamav-scan/0.1/clamav-scan.yaml [hunk 1]
```diff
@@ -96,6 +96,21 @@ spec:
               exit 0
             fi
 
+            timeout=1200  # 20 minutes 
+            interval=60  # interval between checks in seconds
```
Comment (dirgim) [inline task/clamav-scan/0.1/clamav-scan.yaml:100]: Maybe we can lower this interval so we don't end up waiting for up to an extra minute for the database to be ready. Something along 15-20 seconds might work.

Comment (chipspeak) [reply task/clamav-scan/0.1/clamav-scan.yaml:100]: Sure thing!

Comment (chipspeak) [reply task/clamav-scan/0.1/clamav-scan.yaml]: I've kept the interval as 20 seconds and now only output the message to the log every 60 seconds. What do you think?

**Review by dirgim — APPROVED** [review]
LGTM!

**Comment by chipspeak** [comment]
Sample output:
Screenshot from 2024-09-23 10-50-55
Screenshot from 2024-09-23 10-35-36
Screenshot from 2024-09-23 10-18-35

**Review by dheerajodha — APPROVED** [review]
/lgtm

**Review by jencull — COMMENTED** [review]
LGTM

**Pull request merged**